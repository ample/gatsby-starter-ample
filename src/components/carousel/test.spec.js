import React from "react"
import renderer from "react-test-renderer"

import { component as Carousel, fixtures } from "."

describe("Carousel", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Carousel {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
