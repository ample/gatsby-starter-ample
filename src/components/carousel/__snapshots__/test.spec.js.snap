// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel renders correctly 1`] = `
Array [
  <div>
    <button
      id="prev-slide"
    >
      Back
    </button>
    <button
      id="next-slide"
    >
      Next
    </button>
  </div>,
  <div
    className="swiper-container"
  >
    <div
      className="swiper-wrapper"
    >
      <div
        className="swiper-slide"
      >
        Aliquip minim sit non esse mollit elit dolore eiusmod excepteur non.
      </div>
      <div
        className="swiper-slide"
      >
        Ipsum minim sit aliqua commodo velit magna est.
      </div>
      <div
        className="swiper-slide"
      >
        Proident nulla magna ex elit dolore ex ullamco nisi consequat et.
      </div>
      <div
        className="swiper-slide"
      >
        Anim aliqua pariatur dolore reprehenderit nostrud.
      </div>
    </div>
  </div>,
]
`;
